packages/plugins/jwt/src/index.ts

```diff
@@ -0,0 +1,184 @@
+import { JwksClient } from 'jwks-rsa'
+import jsonwebtoken, { JwtPayload, Algorithm } from 'jsonwebtoken'
+import { Plugin, createGraphQLError } from 'graphql-yoga'
+
+const { decode } = jsonwebtoken
+
+export interface JwtPluginOptions {
+  /**
+   * The endpoint to fetch keys from.
+   *
+   * For example: https://example.com/.well-known/jwks.json
+   */
+  jwksUri?: string
+  /**
+   * Signing key to be used to verify the token
+   * You can also use the jwks option to fetch the key from a JWKS endpoint
+   */
+  signingKey?: string
+  /**
+   * List of the algorithms used to verify the token
+   *
+   * Default: ["HS256"] (Allowed values: HS256, HS384, HS512, RS256, RS384, RS512, ES256, ES384, ES512, PS256, PS384, PS512, none)
```

Comment:
```suggestion
   * Default: ["RS256"] (Allowed values: HS256, HS384, HS512, RS256, RS384, RS512, ES256, ES384, ES512, PS256, PS384, PS512, none)
```
Why did we change this default? I guess RS256 is more common, no?

Reply:
Because it is in fact the default signing option from pretty much every JSON library. Probably because it is more simple and doesn't rely on a private/public pair key you have to generate

Reply:
Hmm it is interesting because in many popular libraries and services I always see RS256 is used.

Reply:
For example, in the library used in this plugin (jsonwebtoken) the default is `HS256` :-)

What do I do ? You to restore to `RS256` ?

Reply:
Rs256 feels like a better default.